static/app/stores/tagStore.tsx

```diff
@@ -84,7 +84,7 @@ const storeConfig: TagStoreDefinition = {
       [FieldKey.ISSUE_TYPE]: {
         key: FieldKey.ISSUE_TYPE,
         name: 'Issue Type',
-        values: [],
+        values: ['n_plus_one'],
```

Comment:
Did you mean
```suggestion
        values: ['n_plus_one_db'],
```
? That's the one we're detecting right now

Reply:
hmmmmmmmmm

Reply:
It actually might be _both_ things, 🤔 `performance_n_plus_one_db`

Reply:
`performance_n_plus_one_db_queries` is what it actually is 🤔 🤔 🤔 🤔 🤔